manifests/mongos/params.pp

```diff
@@ -43,6 +46,7 @@
         $unixsocketprefix = '/var/run/mongodb'
         $logpath          = '/var/log/mongodb/mongos.log'
         $fork             = true
+        $service_template = 'mongodb/mongos/RedHat/mongos.service-dropin.epp'
```

Comment:
Maybe that should be a template since there aren't any arguments.

Reply:
You mean an ERB ? I've been told to use EPP now.

Reply:
Ignore that :smile: